tools/start_server.py

```diff
@@ -500,8 +500,8 @@ def uninstall_app(self, _app_id):
 
 def main():
   parser =  argparse.ArgumentParser(description="Handle backend operations for the hmi")
-  parser.add_argument('--host', type=str, required=True, help="Backend server hostname")
-  parser.add_argument('--ws-port', type=int, required=True, help="Backend server port number")
+  parser.add_argument('--host', type=str, default="127.0.0.1", help="Backend server hostname")
```

Comment:
@ValeriiMalkov one more thing here. As you have default values defined in `Flags()` class, its better to use them here as well like
```suggestion
  parser.add_argument('--host', type=str, default=FLAGS.HOST, help="Backend server hostname")
  parser.add_argument('--host', type=str, default=FLAGS.WS_PORT, help="Backend server hostname")
  ...
```